python-regression/README.md

```diff
@@ -1,33 +1,140 @@
+these transactions are resolved, the same approach is used to ensure that value transactions are also being confirmed 
+correctly.  
+
+Machine 3 - Blowball Tests: This machine uses 6 nodes by default, but can be customized to be performed on as many/few 
+as desired. 1000 `getTransactionsToApprove` calls are made across these nodes, and the responses checked to make sure 
+that less than 5% of the results are milestone transactions. If the responses are over this threshold, then that means 
+blowballs are occurring.  
+
+Machine 4 - Stitching Tests: This machine uses 1 node. The node is loaded with a db containing a large side tangle. A 
+stitching transaction is issued, and another transaction referencing that one is issued. After these transactions are 
+issued, making `getTransactionsToApprove` calls should not crash the node.  
+
+Machine 5 - Milestone Validation Tests: This machine uses 2 nodes. Both nodes are loaded with the same db. The db 
+contains several signed milestone transactions, and several unsigned transactions. One node is set to validate the 
+testnet coordinator signature, while the other is not. The one that requires validation should solidify to one point, 
+while the other should solidify further. 
+
+Machine 6 - Local Snapshotting Tests: This machine uses 4 nodes. The first node contains the snapshot `meta` and `state`
+files, the `spent-addresses-db` and `testnetdb` of a synced node. The second only contains the database, and the third 
+only contains the snapshot files. All three of these nodes are tested to ensure that they solidify to the same point, 
+and that the proper information is contained in the snapshot files and databases. The fourth node has a larger database 
+that contains more milestones than the `local-snapshots-pruning-depth`. This node is checked to make sure that after 
+starting, transactions that should be pruned form the database have been pruned correctly.   
+
+### Running Tests on TIAB (Requires present kubernetes configuration)
+These tests have been written to interact with a set of nodes with provided configurations, running on a `kubernetes` 
+cluster. The `ciglue.sh` script will run all the available machines in the `./tests/features/` directory. The nodes will
+be configured based off of the `./tests/features/machine[x]/config.yml` file, and will update the 
+`./tests/features/machine[x]/output.yml` file with the newly created cluster host and port information. This script then
+proceeds to run each of the feature files in the respective machine directories, and these tests are run on the nodes 
+specified in the `output.yml` file. Once the tests have completed, the cluster is torn down. The default logging 
+of these tests is verbose, for debugging purposes. To use this automated script, simply use the following command: 
+```
+bash ciglue.sh [insert iri docker image here]
+```
+
+To run on the latest IOTA build, run:
+```
+bash ciglue.sh iotacafe/iri-dev:latest
+```
```

Comment:
can this work locally just like this without running minikube?
for me it doesn't work...

Anyhows, I don't think it makes sense to invest time in running minikube instructions unless we figure out how to run a subset of tests on that and see how we can debug them.

Make sure you leave a note that the tests will fail if you run them locally like that (unless I am mistaken)